scripts/Inelastic/Direct/RunDescriptor.py

```diff
@@ -456,6 +456,9 @@ def __set__(self,instance,value):
                     self._set_run_list(instance,run_num,file_path,fext)
                 else:
                     self._set_single_run(instance,run_num,file_path,fext)
+        elif isinstance(value, collections.Iterable): # xrange provided
```

Comment:
Just forcing a list to be a list is OK, so you won't need to create a separate case